File: backend-fastify/schemas/document.ts
import { pgTable, serial, text, timestamp } from "drizzle-orm/pg-core";

export const documentTable = pgTable('document', {
  id: serial('id').primaryKey(),
  title: text('title').notNull().unique(),
  description: text('description').notNull(),
  fileName: text('file_name').notNull(),
  createdAt: timestamp('createdAt', { withTimezone: true }).notNull().defaultNow()
})

File: BackendAspNet/BackendAspNet/Document.cs
using System.ComponentModel.DataAnnotations;

namespace BackendAspNet
{
    public class Document
    {
        [Key]
        public int Id { get; set; }

        [Required]
        public string Title { get; set; }

        [Required]
        public string Description { get; set; }

        [Required]
        public string FileName { get; set; }

        public string? File { get; set; }
    }
}